single-post.php

```diff
@@ -1,3 +1,7 @@
+<?php
+$response = wp_remote_get( 'https://wwwtest.ucf.edu/research/wp-json/wp/v2/posts' );
```

Comment:
Debugging code? We need to remove this or else find a way not to hard code the URL if it's intended.

Reply:
I'm not sure why it's over there. Let me see why it's over there.